pkg/infra/pulumi_aws/deploylib.ts

```diff
@@ -1429,7 +1443,7 @@ export class CloudCCLib {
                 ...baseArgs,
                 image: image,
                 portMappings:
-                    nlb.listeners != undefined
+                    nlb != undefined
                         ? nlb.listeners
```

Comment:
minor: If you wanted, I think you could just do this as `nlb?.listeners ?? [ { containerPort ... } ]`